core/src/test/kotlin/io/sharing/server/core/product/domain/ProductTest.kt

```diff
@@ -50,6 +53,52 @@ class ProductTest {
             "image6", "image7", "image8", "image9", "image10"
         )
     }
+
+    @Test
+    fun `등록된 상품을 승인을 할 수 있다`() {
+        val product = createProduct().apply {
+            this.status = ProductStatus.REGISTERED
+        }
```

Comment:
```suggestion
        val product = createProduct(status = REGISTERED)
```

픽스처에 상태도 파라미터로 추가하셨다면, 이용하시는게 더 좋을거같습니다.